src/pilot/assign_role.py

```diff
@@ -428,7 +429,7 @@ def define_storage_logical_disks(drac_client, raid_controller_name):
     #
     # A successful call returns a list, which may be empty; otherwise,
     # None is returned.
-    jbod_capable = is_jbod_capable(drac_client, raid_controller_name)
+    jbod_capable = raid.is_jbod_capable(drac_client, raid_controller_name)
```

Comment:
We should be going through the client object here instead of calling the method in the resource class.  Like this:
drac_client.is_jbod_capable(raid_controller_name)

Reply:
copy

Reply:
done